aspnetcore/fundamentals/index.md

```diff
@@ -461,7 +439,7 @@ An implementation of `IHttpClientFactory` is available for creating `HttpClient`
 * Provides a central location for naming and configuring logical `HttpClient` instances. For example, a *github* client can be registered and configured to access GitHub. A default client can be registered for other purposes.
 * Supports registration and chaining of multiple delegating handlers to build an outgoing request middleware pipeline. This pattern is similar to the inbound middleware pipeline in ASP.NET Core. The pattern provides a mechanism to manage cross-cutting concerns around HTTP requests, including caching, error handling, serialization, and logging.
 * Integrates with *Polly*, a popular third-party library for transient fault handling.
-* Manages the pooling and lifetime of underlying `HttpClientMessageHandler` instances to avoid common DNS problems that occur when manually managing `HttpClient` lifetimes.
+* Manages the pooling and lifetime of underlying `HttpClientHandler` instances to avoid common DNS problems that occur when manually managing `HttpClient` lifetimes.
```

Comment:
I wouldn't normally change content in a < 3.0 version during an update, but I think this one's worth fixing.